sros2/sros2/verb/generate_policy.py

```diff
@@ -125,23 +124,63 @@ def main(self, *, args):
 
             for node_name in node_names:
                 profile = self.get_profile(policy, node_name)
-                subscribe_topics = get_subscriber_info(node=node, node_name=node_name)
+                subscribe_topics = _get_subscriber_info(node=node, node_name=node_name)
                 if subscribe_topics:
                     self.add_permission(
                         profile, 'topic', 'subscribe', 'ALLOW', subscribe_topics, node_name)
-                publish_topics = get_publisher_info(node=node, node_name=node_name)
+                publish_topics = _get_publisher_info(node=node, node_name=node_name)
                 if publish_topics:
                     self.add_permission(
                         profile, 'topic', 'publish', 'ALLOW', publish_topics, node_name)
-                reply_services = get_service_info(node=node, node_name=node_name)
+                reply_services = _get_service_info(node=node, node_name=node_name)
                 if reply_services:
                     self.add_permission(
                         profile, 'service', 'reply', 'ALLOW', reply_services, node_name)
-                request_services = get_client_info(node=node, node_name=node_name)
+                request_services = _get_client_info(node=node, node_name=node_name)
                 if request_services:
                     self.add_permission(
                         profile, 'service', 'request', 'ALLOW', request_services, node_name)
 
         with open(args.POLICY_FILE_PATH, 'w') as stream:
             dump_policy(policy, stream)
         return 0
+
+
+def _get_node_names(*, node, include_hidden_nodes=False):
```

Comment:
I don't think these are more generally useful. They're very specific to this verb. Other packages will consume node names/namespaces on their own.

Reply:
Agreed